docs/_config.yml

```diff
@@ -123,3 +123,9 @@ es:
     - path: es/week01/01.md
       sections:
         - path: es/week01/01-1.md
+
+it:
+  title:   'Apprendimento Profondo'
+  tagline: 'Gotta learn them all in Italian'
```

Comment:
This was actually useless, can you delete it?

Reply:
Ooooh you are the webmaster for the course! Excellent to be working with you again 💃 

Will do :)

Reply:
:dancer:

Reply:
What's going on here? You know each other?!

Reply:
Alright, removed -- @ebetica where / how are you translating the title now for each language?

Reply:
Just the `tagline`, not everything! Haha!

Reply:
Going to cook some lunch, then I'll merge this!
I can't wait to have an Italian version up!

Reply:
>Just the tagline, not everything! Haha!

Oh noes -- sigh -- I'll revert the reverting.

>What's going on here? You know each other?!

Fellow dancers of the tensors 💃(ask Zeming directly :P )

@Atcold hold up merging / reviewing this, let me clean it up first :)